src/components/settings/SettingsLMSTab/tests/BlackboardConfig.test.jsx

```diff
@@ -247,4 +252,26 @@ describe('<BlackboardConfig />', () => {
     expect(window.open).toHaveBeenCalled();
     expect(mockFetchSingleConfig).toHaveBeenCalledWith(1);
   });
+  test('validates poorly formatted existing data on load', async () => {
```

Comment:
do we need async here? I did not see any awaits

Reply:
I thiiink I just accidentally left the async in haha